README.md

```diff
@@ -90,6 +90,38 @@ with session.logger(dataset_name="my_dataset") as logger:
     #images
     logger.log_images("path/to/image.png")
 ```
+
```

Comment:
Add a section for visualizing the profile here

Reply:
done

Reply:
I think we should move the paragraph starting with "whylogs collects approximate statistics and sketches..." right after the first code example